server/routes/api/vote.js

```diff
@@ -0,0 +1,29 @@
+import { Router } from 'express';
+import AricleValidator from '../../middlewares/validations/articleValidator';
+import VoteController from '../../controllers/votes/VoteController';
+import JWTAuthentication  from '../../middlewares/JWTAuthentication';
+
+const voteRouter = Router();
+
+voteRouter.post(
+  '/votes/:articleId/like',
+  JWTAuthentication.authenticateUser,
+  AricleValidator.articleExistInDatabase,
+  VoteController.likeAnArticle,
+);
+
+voteRouter.put(
+  '/votes/:articleId/unlike',
```

Comment:
Difference between "Unlike" and "Dislike" is not clear on the surface.

Reply:
Okay so the unlike is used to take the vote state to its default `0` while dislike show that you dislike an article. This situation might arise when a user like an article by mistake and wants to revert.